http-core/src/main/scala/org/apache/pekko/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -404,9 +407,18 @@ private[http] object HttpServerBluePrint {
     import pekko.http.impl.util.JavaMapping.Implicits._
 
     /** JAVA API * */
-    def update(timeout: Duration, handler: Function[model.HttpRequest, model.HttpResponse]): Unit =
+    override def updateTimeout(timeout: java.time.Duration): Unit = {
+      val stimeout = if (timeout eq null) null else timeout.asScala
```

Comment:
Not sure why timeout can be `null`.

Reply:
the existing code passes around nulls internally unfortunately